packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 51.2.0 - February 24, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Add namespace support to test:run & debug commands ([PR #2961](https://github.com/forcedotcom/salesforcedx-vscode/pull/2961))
```

Comment:
Let's add a link to the issue that was filed for this one -https://github.com/forcedotcom/salesforcedx-vscode/issues/2865
And maybe something like "Add namespace support for running and debugging tests" for the text